src/main/java/redis/clients/jedis/StreamEntryID.java

```diff
@@ -151,4 +172,20 @@ public String toString() {
       return "+";
     }
   };
+
+  /**
+   * Should be used only with XREAD
+   *
+   * {@code XREAD STREAMS mystream +}
+   */
+  // TODO: unify with MAXIMUM_ID ??
+  public static final StreamEntryID XREAD_LAST_ENTRY = new StreamEntryID() {
```

Comment:
This could be `XREAD_LAST_ENTRY = MAXIMUM_ID`, to make it obvious that it is only an alias. Somehow I find it more informative to have different names, depending on the usage context.